package.json

```
@@ -22,27 +22,27 @@
     "@dicebear/api-8": "*",
     "@dicebear/api-9": "*",
     "@dicebear/converter": "^9.2.2",
-    "@fastify/cors": "^9.0.1",
+    "@fastify/cors": "^9",
     "change-case": "^5.4.4",
-    "fastify": "^4.28.1",
-    "qs": "^6.13.0"
+    "fastify": "^4",
```

This would allow a downgrade. We should avoid that.

```suggestion
    "fastify": "^4.29.1",
```

---

Fixed in 03efcb936703dc9b2ccc6f6ebcaa879f72fa9392.